tests/fe/interpolate_system_2.cc

```diff
@@ -30,10 +30,10 @@
 template <int dim, typename T>
 void check(T function, const unsigned int degree)
 {
-  auto fe = FESystem<dim>(FE_RaviartThomas<dim>(degree),
-                          2,
-                          FESystem<dim>(FE_RaviartThomas<dim>(degree), 2),
-                          1);
+  FESystem<dim> fe (FE_RaviartThomas<dim>(degree),
+                    2,
+                    FESystem<dim>(FE_RaviartThomas<dim>(degree), 2),
+                    1);
```

Comment:
I am a bit surprised that gcc-7 accepted above code. Sorry for the inconvenience.